pages/task.tsx

```diff
@@ -0,0 +1,26 @@
+import { NextPage } from 'next'
+import styles from '../styles/assessment.module.css'
+import data from '../data/IT2810Høst2018.json'
+import Header from "../components/header";
+import {Button} from "@mui/material";
+
+const Approval: NextPage = () => {
+
+    const totalTasks: number = data.ext_inspera_candidates[0].result.ext_inspera_questions.length;
+    const taskNumbers: number[] = Array.from(Array(totalTasks).keys())
+
+  
+
+  return (
+    <div className={styles.container}>
+        <Header data={data}/> 
+      <main>
+          {taskNumbers.map(
```

Comment:
I think we can use MUI Grid for this, but that can be a later issue. (to make the page pretty)